Content.Shared/CartridgeLoader/Cartridges/SpaceVendorsUiState.cs

```diff
@@ -0,0 +1,37 @@
+using Robust.Shared.Serialization;
+
+namespace Content.Shared.CartridgeLoader.Cartridges;
+
+[Serializable, NetSerializable]
+public sealed class SpaceVendorsUiState : BoundUserInterfaceState
+{
+    public List<AppraisedItem> AppraisedItems;
+
+    public SpaceVendorsUiState(List<AppraisedItem> appraisedItems)
+    {
+        AppraisedItems = appraisedItems;
+    }
+}
+
+[Serializable, NetSerializable, DataRecord]
+public sealed class AppraisedItem
+{
+    public readonly string Name;
+    public readonly string Price;
+    public int Minutes = 0;
+
+    private readonly DateTime _dateTimeCreation;
+
+    public AppraisedItem(string name, string price)
+    {
+        Name = name;
+        Price = price;
+        _dateTimeCreation = DateTime.Now;
+    }
+
+    public void UpdateElapsedTimeData()
+    {
+        TimeSpan date = DateTime.Now - _dateTimeCreation;
+        Minutes = date.Hours * 60 + date.Minutes;
+    }
+}
```

Comment:
Only the DateTime is needed when the item is created.

Reply:
Do you want me to remove the assignments to the Name and Price variables from the constructor, and assign only datetime?

Reply:
but then how do I pass the Name and Price to the UI?

Reply:
Name and price are okay but the datetime should be the servertime when it's created, don't use DateTime.Now use IGameTiming.CurTime.